microservices/password-manager/utils/command-utils.ts

```diff
@@ -128,13 +133,13 @@ const runCommandInPipe = (
   }
 
   const outputData = readFileSync(outputPath).toString();
-  return new CommandOutput(CommandOutputType.Success, outputData);
+  return new CommandOutput("success", outputData);
 };
 
-const runScriptInPipe = (scriptFileName, password = null, args = []) => {
+const runScriptInPipe = (scriptFileName: string, password: string = null, args?: string[]) => {
   const scriptPath = path.join("scripts", scriptFileName);
-  const copyScriptResult = runCommand(`cp ${scriptPath} ${PIPE_COMM_DIR}`);
-  if (copyScriptResult.type === CommandOutputType.Error) {
+  const copyScriptResult = new CommandExecutor(`cp ${scriptPath} ${PIPE_COMM_DIR}`, new PipeExecuteStrategy()).execute();
```

Comment:
PipeExecutorStrategy is built only using a builder, it should not be built directly.